docs/overview.md

```diff
@@ -41,19 +39,22 @@ Presidio uses OCR to detect text in images. It further allows the redaction of t
 Presidio accepts multiple sources and targets for data annonymization. Specifically:
 
 1. Storage solutions
-    * Azure Blob Storage
-    * S3
-    * Google Cloud Storage
+
+   - Azure Blob Storage
+   - S3
+   - Google Cloud Storage
```

Comment:
Could you please add Azure Data Lake gen 2?

Reply:
done